assets/src/application/services/utilities/text/entityListSearch/index.test.ts

```diff
@@ -0,0 +1,78 @@
+import { Entity } from '@appServices/apollo/types';
+import entityListSearch from './';
+import { EntityListSearchProps } from './types';
+
+// Create a test entity to add more fields
+interface TestEntity extends Entity {
+	description: string;
+}
+
+const entities: TestEntity[] = [
+	{
+		id: 'PGF0ZXRpbWU6ODM=',
+		dbId: 83,
+		name: 'test',
+		description: 'some desc',
+	},
+	{
+		id: 'QGF0ZXRpbWU6ODU=',
+		dbId: 85,
+		name: 'just another entity',
+		description: 'test desc',
+	},
+	{
+		id: 'RGF0ZXRpbWU6ODU=',
+		dbId: 86,
+		name: 'another title',
+		description: null,
+	},
+];
+
+const searhArgs: EntityListSearchProps<TestEntity> = {
+	entities,
+	searchFields: ['name', 'description'],
+	searchText: '',
+};
+
+describe('entityListSearch', () => {
+	it('should return two results by name and description', () => {
```

Comment:
`description` is being mentioned, but is not checked in the assertion

Reply:
It it used because description contains that keyword